codegen-core/src/main/kotlin/software/amazon/smithy/rust/codegen/core/smithy/protocols/parse/EventStreamUnmarshallerGenerator.kt

```diff
@@ -43,6 +46,15 @@ import software.amazon.smithy.rust.codegen.core.util.expectTrait
 import software.amazon.smithy.rust.codegen.core.util.hasTrait
 import software.amazon.smithy.rust.codegen.core.util.toPascalCase
 
+fun RustModule.Companion.eventStreamSerdeModule(): RustModule.LeafModule =
+    new(
+        "event_stream_serde",
+        visibility = Visibility.PUBLIC,
```

Comment:
Seems like this was only made public to support tests? It definitely shouldn't actually be public as it will cause customer confusion and tie our hands with backwards compatibility requirements.

Reply:
Thank you, I've made this private again after making the tests unit tests instead of integration tests.